src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -2150,7 +2156,7 @@ def create_sync_pipe(self, group_id: str, pipe_id: str,
     def remove_sync_pipe(self, group_id: str, pipe_id: str,
                          source_zones: Optional[List[str]] = None,
                          destination_zones: Optional[List[str]] = None,
-                         destination_buckets: Optional[List[str]] = None, bucket_name: str = ''):
+                         destination_bucket: Optional[List[str]] = None, bucket_name: str = ''):
```

Comment:
The change is it to restrict it to a string rather than a list, right?
Same as you've done on the `create_sync_pipe`

```suggestion
                         destination_bucket: str = '', bucket_name: str = ''):
```

Reply:
yes, maybe this got overwritten in the conflict resolution,
will fix it, thanks